charts/kafka-operator/templates/crds.yaml

```diff
@@ -4143,6 +4143,11 @@ spec:
                             type: string
                         type: object
                       type: array
+                    virtualHostName:
```

Comment:
I think the CRD should be updated since this is a map in the current implementation (no longer a string)